habitat_baselines/config/pointnav/ppo.yaml

```diff
@@ -0,0 +1,40 @@
+TRAINER:
+  TRAINER_NAME: "ppo"
+  RL:
+    PPO:
+      # general options
+      tensorboard_dir: "tb"
+      num_processes: 1
+      log_interval: 10
+      pth_gpu_id: 0
+      sim_gpu_id: 0
+      checkpoint_interval: 50
+      checkpoint_folder: "data/new_checkpoints"
+      task_config: "configs/tasks/pointnav.yaml"
+      num_updates: 10000
+      # eval specific:
+      count_test_episodes: 2
+      video_option: ["disk", "tensorboard"]
+      video_dir: "video_dir"
+      model_path: "data/checkpoints/ckpt.123.pth"
+      tracking_model_dir: ""
+      # 1 of the above 2 options needs to be empty
+
+      # ppo params
+      clip_param: 0.1
+      ppo_epoch: 4
+      num_mini_batch: 1
+      value_loss_coef: 0.5
+      entropy_coef: 0.01
+      lr: 2.5e-4
+      eps: 1e-5
+      max_grad_norm: 0.5
+      num_steps: 128
+      hidden_size: 512
+      use_gae: True
+      gamma: 0.99
+      tau: 0.95
+      use_linear_clip_decay: True
+      use_linear_lr_decay: True
+      reward_window_size: 50
+      sensors: "RGB_SENSOR,DEPTH_SENSOR"
```

Comment:
This should be a list

Reply:
I think this not being a list originally is to support the command line option augmentation [feature](https://github.com/facebookresearch/habitat-api/blob/732377878c8c4b2503358e184bac4b17f2a8937e/habitat/config/default.py#L188). I kept the feature to add config option via command line so maybe a string is better suited here.

Reply:
The `video_option` field is a list tho?  Is that not able to be changed and should be a comma separated string?

Reply:
I think that should also be a string instead for consistency. Will fix :)

Reply:
Agree with Erik it should be list, but is in scope of next RL refactoring PR.

Reply:
Command line arguments from yacs support list since it does literal eval on what you pass. So this can be converted to an array.

Reply:
@apsdehal Thanks for pointing out! I didn't test `option '["a","b"]'` and turns out that works for literal eval. Updated [here](https://github.com/facebookresearch/habitat-api/pull/172/commits/bba7ac81591d447eb31be2623e6c69b9fcec6840) in #172.